quinn-udp/src/unix.rs

```diff
@@ -568,6 +753,17 @@ fn prepare_msg(
     encoder.finish();
 }
 
+#[cfg(all(
+    not(feature = "fast-apple-datapath"),
+    any(
+        target_os = "macos",
+        target_os = "ios",
+        target_os = "tvos",
+        target_os = "visionos",
+        target_os = "openbsd",
+        target_os = "netbsd"
```

Comment:
```suggestion
        target_os = "netbsd",
        target_os = "linux",
```

Also, what about solaris?